.github/workflows/ci.yml

```
@@ -3,9 +3,9 @@ on:
   push:
     branches:
       - main
-  pull_request:
-  merge_group:
-  workflow_dispatch:
+  # pull_request:
+  # merge_group:
+  # workflow_dispatch:
```

This has to be undone. I did this because I didn't want to block the CI.

```suggestion
  pull_request:
  merge_group:
  workflow_dispatch:
```